data-generation/generators/generateRandomDate.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable camelcase */
+const moment = require('moment');
+const { extendMoment } = require('moment-range');
+
+// eslint-disable-next-line no-unused-vars
+const momentRange = extendMoment(moment);
+
+// eslint-disable-next-line no-unused-vars
+function generateRandomDateWithinLastThreeMonths() {
```

Comment:
This function is larger than the script :). However, you have the most accurate date info.